<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightSystem</class>
 <widget class="QDialog" name="LightSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#LightSystem{
background-color:rgb(178,178,178,255);
}</string>
  </property>
  <widget class="QFrame" name="bottom">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>491</width>
     <height>811</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#bottom
{
	background-color:#ffffff;
 
	border:none;
	border-radius:25px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QStackedWidget" name="PageBottom">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>491</width>
      <height>811</height>
     </rect>
    </property>
    <widget class="QWidget" name="LightMain">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>320</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#282A49</string>
      </property>
      <property name="text">
       <string>室内亮度</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="BrightnessP2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>280</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="CurtainNumber">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>351</width>
        <height>321</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>PibotoLt</family>
        <pointsize>150</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>100</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="ButtonBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>380</y>
        <width>331</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox#ButtonBox{
border:none;
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="AllLed_Status">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>291</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Adobe 黑体 Std R</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border-radius:15px;
	background:#FFFFFF;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../other/main.qrc">
         <normaloff>:/new/Led/Led/All_Light_OFF.png</normaloff>
         <normalon>:/new/Led/Led/AirSwitch-ON.png</normalon>:/new/Led/Led/All_Light_OFF.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="AiMode">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>291</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Adobe 黑体 Std R</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border-radius:15px;
	background:#FFFFFF;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../other/main.qrc">
         <normaloff>:/new/Led/Led/AI_ON.png</normaloff>
         <normalon>:/new/Led/Led/AI_OFF.png</normalon>:/new/Led/Led/AI_ON.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="autoRepeatDelay">
        <number>300</number>
       </property>
       <property name="autoRepeatInterval">
        <number>200</number>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="BackMain">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">max-width:50px;
 max-height:50px;
 min-width:50px;
 min-height:50px;
border-radius:25px;
background-color: white;
 border-style: solid;</string>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ModeChoice_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>750</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">max-width:50px;
 max-height:50px;
 min-width:50px;
 min-height:50px;
border-radius:25px;
background-color: white;
 border-style: solid;</string>
      </property>
      <property name="text">
       <string>Mode</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AloneControl">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>750</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">max-width:50px;
 max-height:50px;
 min-width:50px;
 min-height:50px;
border-radius:25px;
background-color: white;
 border-style: solid;</string>
      </property>
      <property name="text">
       <string>N</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="AloneLight">
     <widget class="SwitchButton" name="Led_Switch" native="true">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>760</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="Brightness_Slider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>510</y>
        <width>351</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="XProgressBar" name="Brightness_Xprogress" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>435</y>
        <width>441</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="Color_Slider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>630</y>
        <width>351</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="XProgressBar" name="Color_Xprogress" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>560</y>
        <width>441</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="InsidesBrightness">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>371</width>
        <height>241</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>PibotoLt</family>
        <pointsize>150</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
border-radius:15px;</string>
      </property>
      <property name="text">
       <string>832</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>340</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#282A49</string>
      </property>
      <property name="text">
       <string>室内亮度</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSlider" name="Device_Slider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>720</y>
        <width>441</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>6</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>680</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#282A49</string>
      </property>
      <property name="text">
       <string>当前灯光:</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>460</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#282A49</string>
      </property>
      <property name="text">
       <string>亮度：</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Brightness_Text">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>460</y>
        <width>31</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#282A49</string>
      </property>
      <property name="text">
       <string>80</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="SwitchButton" name="Lux_All" native="true">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>500</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="SwitchButton" name="Color_All" native="true">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>620</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="BrightnessP">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>320</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>670</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ModeChoice">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>20</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">max-width:50px;
 max-height:50px;
 min-width:50px;
 min-height:50px;
border-radius:25px;
background-color: white;
 border-style: solid;</string>
      </property>
      <property name="text">
       <string>Main</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>590</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#282A49</string>
      </property>
      <property name="text">
       <string>色温：</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Color_Text">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>590</y>
        <width>31</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#282A49</string>
      </property>
      <property name="text">
       <string>80</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="SelectedLed">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>675</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#282A49</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header>switchbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XProgressBar</class>
   <extends>QWidget</extends>
   <header>xprogressbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../other/main.qrc"/>
 </resources>
 <connections/>
</ui>
